proto/fairyring/pep/tx.proto

```diff
@@ -90,4 +93,25 @@ message MsgGetGeneralKeyshare {
   string req_id  = 2;
 }
 
-message MsgGetGeneralKeyshareResponse {}
\ No newline at end of file
+message MsgGetGeneralKeyshareResponse {}
+
+message MsgRequestPrivateIdentity {
+  option (cosmos.msg.v1.signer) = "creator";
+  string                   creator = 1;
+  string                   reqId   = 2;
+  cosmos.base.v1beta1.Coin amount  = 3 [(gogoproto.nullable) = false];
+}
+
+message MsgRequestPrivateIdentityResponse {
+  string req_id = 1;
+}
+
+message MsgGetPrivateKeyshares {
+  option (cosmos.msg.v1.signer) = "creator";
+  string creator       = 1;
+  string reqId         = 2;
+  string rsa_64_pubkey = 3;
```

Comment:
should change to just the `secp256k1` public key

Reply:
done